File: ocel/src/main/java/it/unibz/inf/pm/ocel/validation/Validate.java
package it.unibz.inf.pm.ocel.validation;


import org.dom4j.DocumentException;

import java.io.IOException;

public class Validate {
    public static boolean apply(String input_path, String validation_path, String parameters) throws DocumentException, IOException {
        if (input_path.indexOf(".json") != -1){
            return Oceljson.apply(input_path, validation_path, parameters);
        }else{
            return Ocelxml.apply(input_path,validation_path, parameters);
        }
    }
}

File: pyocel/ocel/validation/validate.py
from ocel.validation import oceljson, ocelxml


def apply(input_path, validation_path, parameters=None):
    if ".json" in input_path:
        return oceljson.apply(input_path, validation_path, parameters=parameters)
    else:
        return ocelxml.apply(input_path, validation_path, parameters=parameters)
